domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/note/dto/UpdateNoteDto.kt

```diff
@@ -4,6 +4,6 @@ import java.util.UUID
 
 data class UpdateNoteDto(
   val reference: UUID,
-  val content: String?,
+  val content: String,
```

Comment:
A note's content cannot be null - you shouldnt be able to update a note with null content 👍